portfolio/src/main/webapp/index.html

```diff
@@ -4,28 +4,46 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    
   </head>
   <body>
-    <div id="content">
+    <div class="tab">
+        <button id="defaultOpen" class="tablinks" onclick="openTab(event, 'about-me')">About me</button>
+        <button class="tablinks" onclick="openTab(event, 'blog')">NFT Blog</button>
+        <button class="tablinks" onclick="openTab(event, 'terminal')">Terminal</button>
+    </div>
+    <div id="about-me" class = "tabcontent">
       <h1>Mick Toyingsirikul</h1>
       <img src="./images/doodle.png" alt = "Avatar picture"/>
       <p>Hello 👋! My name is Mick, I'm a 4th year CS student at the University of British Columbia.</p>
       <p>My interests are Computer Systems, Algorithms, and Database Systems.  Apart from school, I'm a stock investor.  
           Recently, I start trading/learning crypto and NFT.</p>
       <p>Currently, I'm looking for a Software Engineer internship.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
       <p>Contact Me</p>
```

Comment:
Maybe change the font of this? It matches the style of the rest of your text, but it's sort of like a separate "Contact" section. 

Another option is to just make this a sentence: "Connect with me on Github (link) or LinkedIn (link)". 

Another other option is to make the contact a separate tab.

Reply:
I make this into a sentence.